Source/Core/DolphinQt2/GameList/TableDelegate.cpp

```diff
@@ -52,24 +58,44 @@ void TableDelegate::paint(QPainter* painter, const QStyleOptionViewItem& option,
 
 QSize TableDelegate::sizeHint(const QStyleOptionViewItem& option, const QModelIndex& index) const
 {
+  const auto scale = DPI::GetScale();
   switch (index.column())
   {
   case GameListModel::COL_PLATFORM:
-    return Resources::GetPlatform(0).size();
+    return Resources::GetPlatform(0).size() / Resources::GetIconScale() * scale;
   case GameListModel::COL_COUNTRY:
-    return Resources::GetCountry(0).size();
+    return Resources::GetCountry(0).size() / Resources::GetIconScale() * scale;
   case GameListModel::COL_RATING:
-    return Resources::GetRating(0).size();
+    return Resources::GetRating(0).size() * scale;
   case GameListModel::COL_BANNER:
-    return NORMAL_BANNER_SIZE;
+    return NORMAL_BANNER_SIZE * scale;
```

Comment:
Don't multiply this by `scale`. With `AA_EnableHighDpiScaling` on, just `NORMAL_BANNER_SIZE` is correct.